packages/thirdweb/package.json

```diff
@@ -402,6 +402,9 @@
       ],
       "wallets/*": [
         "./dist/types/exports/wallets/*.d.ts"
+      ],
+      "insight": [
+        "./dist/types/exports/insight.d.ts"
```

Comment:
didnt know we had to do this. good to know

Reply:
oh shit yeah we do